testing/unit/tp/node_account/test_transfer_from_unfrozne_to_operational.py

```diff
@@ -0,0 +1,136 @@
+"""
+Provide tests for node account transferring from unfrozen to operational
```

Comment:
Use dots at the end of the sentences in the docstrings.

Reply:
Done.